tests/README.md

```diff
@@ -0,0 +1,3 @@
+# ccftools
```

Comment:
Should this be `tests/ccftools/README.md`?

Reply:
From the different examples I've seen online, the `README.md` is often at the top directory. I agree with you though and I've moved it in there 👍

Reply:
I think we probably ought to move ccftools out of tests?

Reply:
Done. This is now under `python/ccf`.